backend/manager/modules/common/src/main/java/org/ovirt/engine/core/common/businessentities/storage/TransferClientType.java

```diff
@@ -31,11 +31,6 @@ public static TransferClientType forValue(int value) {
         return valueToType.get(value);
     }
 
-    @Override
```

Comment:
maybe it would make sense to somehow discourage anyone from adding it, but I don't really see any reason why anyone would at this point